settings.gradle.kts

```diff
@@ -100,13 +100,14 @@ gradleEnterprise {
 }
 
 gitRepositories {
-  checkoutsDirectory.set(file(rootProject.projectDir.absolutePath + "/hedera-node/hapi/"))
+  checkoutsDirectory.set(File(rootDir, "hedera-node/hapi"))
+  // check branch in repo for updates every 10 minutes
+  refreshIntervalMillis.set(10 * 60 * 1000)
   include("hedera-protobufs") {
     uri.set("https://github.com/hashgraph/hedera-protobufs.git")
     // choose tag or branch of HAPI you would like to test with
     // this looks for a tag in hedera-protobufs repo
-    // This version needs to match tha HAPI version below in versionCatalogs
-    tag.set("add-pbj-types-for-state")
+    branch.set("add-pbj-types-for-state")
```

Comment:
I think this was the problem with the _Git Repo Update_

If you use "tag" here, the plugin does not expect changes. If you use "branch" the plugin should regularly check and update. Then there should be no need to extend/use the "clean" task.